### ISSUE TITLE
excel_sheets fails on path, read_excel works

### ISSUE THREAD

**Issue by ucb** [tidyverse/readxl#556]
File names that contain remote ASCII fail (with "file does not exist" error) on execution of excel_sheets, while read_excel works correctly and without warnings/errors

Example: Norwegian letter "Å"

Temporary workaround: I have changed sys.locale from Rstudio and that did the trick. It would be nice not to have to play with it, especially since read_excel works as is!

NB: `Sys.setlocale("LC_ALL","...")` replacing ... with the language choice that works for you (does not have to be the original language, but the default encoding used by Windows.

**Comment by jennybc** [comment]
This is a case where an actual reprex, including session info and locale and an example sheet, would be very productive. Otherwise, I have little hope of really digging in.

---

## How to provide a readxl reprex

We're in a much better position to address your issue if you can provide a reprex (**repr**oducible **ex**ample). Provide as much of this as you can:

  * An actual xls or xlsx file. Pick one:
    - Your personal xls or xlsx: try to strip it down to the minimal size and complexity to demonstrate your point. And, obviously, remove any sensitive data.
    - A publicly available xls or xlsx: provide URL and the code you used to download.
  * A small bit of R code that uses readxl on the provided xls or xlsx file and demonstrates your point.
    - Consider using the [reprex package](https://github.com/jennybc/reprex#readme) to prepare this. In addition to nice formatting, this ensures your reprex is self-contained.
  * Any details about your environment that seem clearly relevant, such as operating system.
        `reprex(..., si = TRUE)`
    will append a standard summary, folded neatly away, at the bottom of your reprex.

How to provide your own xls/xlsx file? In order of preference:

  1. [Attach the file directly to your issue](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/). Instructions are always at the bottom of the issue or comment box. `.xlsx` is a supported file type. You'll need to zip or gzip `.xls` so it appears as `.zip` or `.gz`.
  1. Share via DropBox or Google Drive and provide the link in your issue.
  1. Explain you absolutely cannot provide a relevant file via github.com and offer to provide privately.
  1. Don't share a file and realize you're hoping for, e.g., a bug fix with no concrete example to go on.

**Comment by panporter** [comment]
I'd like to hook in providing a hopefully useful reprex.

```
library(readxl)

Sys.setlocale("LC_ALL", "English_United States.1252") # not sure if necessary
utf8 <- "\u00FC.xlsx" # LATIN SMALL LETTER U WITH DIAERESIS
latin1 <- iconv(utf8, from="UTF-8", "latin1") # get same filename in different encoding

Encoding(utf8) # UTF-8
Encoding(latin1) # latin1
identical(utf8, latin1, FALSE, FALSE, FALSE, FALSE, FALSE) # is TRUE, quite confusing

readxl::read_excel(utf8) # works
readxl::read_excel(latin1) # works

readxl::read_excel(utf8, sheet="A") # works
readxl::read_excel(latin1, sheet="A") # Evaluation error. zip file 'Ã¼.xlsx' cannot be opened.

# Windows 10

# Version

# platform       x86_64-w64-mingw32          
# arch           x86_64                      
# os             mingw32                     
# system         x86_64, mingw32             
# status                                     
# major          4                           
# minor          0.4                         
# year           2021                        
# month          02                          
# day            15                          
# svn rev        80002                       
# language       R                           
# version.string R version 4.0.4 (2021-02-15)
# nickname       Lost Library Book
```

The file name should be a _LATIN SMALL LETTER U WITH DIAERESIS_. I zipped the file since GitHub seems to translate file names.

[ü.xlsx.zip](https://github.com/tidyverse/readxl/files/6034039/u.xlsx.zip)

**Comment by panporter** [comment]
There seems to be some character set conversion gone wrong, since Ã¼ is the latin1/ISO-8859 representation of 0xC3BC which is indeed UTF-8 representation of ü.

**Comment by panporter** [comment]
#639 seems to be a duplicate or at least a similar issue.

**Comment by panporter** [comment]
Okay, digging into the code reveals: These issues seem to be fixed since 5f210c3c359d58d0f2d4cd45f368e661a9c540d3 but latest release of version 1.3.1 on 2019-03-19 was prior to this fix.

Any plans of a new release?

**Comment by jennybc** [comment]
I am releasing soon, which will put the existing fix into more hands.
